src/main.rs

```diff
@@ -152,8 +152,11 @@ fn main() -> Result<()> {
     let db = nu_command::open_connection_in_memory_custom()?;
     #[cfg(feature = "sqlite")]
     db.last_insert_rowid();
-
-    let (args_to_nushell, script_name, args_to_script) = gather_commandline_args();
+    let commandline_args = gather_commandline_args();
+    let args_to_nushell = commandline_args.args_to_nushell;
+    let script_name = commandline_args.script_name;
+    let args_to_script = commandline_args.args_to_script;
+    let args_to_commands = commandline_args.args_to_commands;
```

Comment:
You can destructure this using a pattern on the left-hand side of the `let`

Reply:
Done